packages/wrangler/e2e/helpers/command.ts

```diff
@@ -40,15 +41,13 @@ export function runCommand(
 			encoding: "utf8",
 			timeout,
 		});
-		// eslint-disable-next-line turbo/no-undeclared-env-vars
-		if (process.env.VITEST_MODE === "WATCH") {
-			if (stdout.length) {
-				console.log(stdout);
-			}
-			if (stderr.length) {
-				console.error(stderr);
-			}
+		if (stdout.length) {
+			console.log(`[${path.basename(cwd ?? "/unknown")}]`, stdout);
 		}
+		if (stderr.length) {
+			console.error(`[${path.basename(cwd ?? "/unknown")}]`, stderr);
+		}
```

Comment:
This means our E2E test output will be a _lot_ noisier, but I think we should keep logging enabled by default for now to help us diagnose and fix flakes